templates/egressfirewall.go

```diff
@@ -29,6 +29,12 @@ var EgressFirewallTemplate = ovnv1.EgressFirewall{
 				},
 				Type: ovnv1.EgressFirewallRuleAllow,
 			},
+			{
```

Comment:
- pls do add egress rules for RHOBS as well if not done already
- @ezio-auditore other than SSO & RHOBS do we need any other egress rules wrt alerting?

Reply:
Done